lib/v2/so/search.js

```diff
@@ -0,0 +1,39 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const renderDescription = (description, images) => art(path.join(__dirname, './templates/description.art'), { description, images });
+
+module.exports = async (ctx) => {
+    const { keyword } = ctx.params;
+    const url = `https://www.so.com/s?q=${encodeURIComponent(keyword)}`;
+    const key = `360-so-search:${url}`;
+    const response = await ctx.cache.tryGet(key, async () => (await got(url)).data);
```

Comment:
- Use the cache efficiently. Do not cache the whole HTML. Cache the processed output instead.

- Please also note that you're calling [`tryGet()`](https://docs.rsshub.app/joinus/advanced/use-cache#ctxcachetrygetkey-getvaluefunc--maxage--refresh-) without disabling `refesh`. The content will be kept extending its expiration time whenever it's called using the same key.